jsonschema-core.xml

```diff
@@ -181,17 +181,14 @@
 
             <section title="JSON Document">
                 <t>
-                    A JSON document is an information resource (series of octets) described by the
-                    application/json media type.
```

Comment:
iiuc this means "JSON documents" := "JSON text", thus resulting in a redundant definition.

Reply:
I think you're right. I don't think there's much value in including that sentence.

Reply:
Damn